specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-04-01-preview/managedHsm.json

```diff
@@ -602,7 +859,238 @@
         }
       },
       "description": "The server error."
-    }
+    },
+    "MHSMNetworkRuleSet": {
+        "properties": {
+          "bypass": {
+            "type": "string",
+            "description": "Tells what traffic can bypass network rules. This can be 'AzureServices' or 'None'.  If not specified the default is 'AzureServices'.",
+            "enum": [
+              "AzureServices",
+              "None"
+            ],
+            "x-ms-enum": {
```

Comment:
Ideally, please provide descriptions for the enum values.

Reply:
Same for enums below. Probably not as important for management plane, but for data plane this helps generate good, consistent documentation across languages.

Reply:
can you please give an example of how to do that? I looked for a few other pecs in resource-manager but could not find.
There is one description field added to it, is that sufficient for management plane?